task/net/Cargo.toml

```diff
@@ -10,12 +10,12 @@ cortex-m = {version = "0.7", features = ["inline-asm"]}
 num-traits = {version = "0.2.12", default-features = false}
 serde = {version = "1", default-features = false, features = ["derive"]}
 ssmarshal = {version = "1", default-features = false}
-stm32h7 = {version = "0.13", features = ["stm32h743"]}
+stm32h7 = {version = "0.13"}
```

Comment:
...and actually it looks like I accidentally omitted `default-features = false` from this line, before you arrived. :-)